frontend/tests/integration/spec_helper.js

```diff
@@ -0,0 +1,7 @@
+var chai = require('chai');
+var chaiAsPromised = require('chai-as-promised');
+chai.use(chaiAsPromised);
+
+module.exports = {
+  expect: chai.expect
+};
```

Comment:
@hirtie-maxim  I always get the chills when there is no newline at the end of a file :laughing: